modules/ROOT/pages/flow-component.adoc

```diff
@@ -45,11 +45,11 @@ up business events. You can also set up error handling for a component
 
 If you set the initial state to Stopped (`initialState="stopped"`), you need to use xref:runtime-manager::flow-management.adoc[Runtime Manager] to activate the flow, or you can simply reset the Flow configuration to `Started`. Note that the console
 output for this state looks something like this: `Flow flow_component_ex has not been started (initial state = 'stopped')`
-| Max Concurrency (`maxConcurrency`) a| Optional. The default is limited only by the availability of threads. The container thread pool determines the maximum level of parallelism that the flow can use to optimize the performance when processing messages. You can use Max Concurrency to prevent the flow from accepting new messages while the configured number of maximum request is being processed.
+| Max Concurrency (`maxConcurrency`) a| Optional. Sets the maximum number of concurrent messages that a flow can process. If not set, the container thread pool determines this value. While the flow is processing the maximum number of concurrent messages, it cannot receive additional requests.
```

Comment:
this is not right. The container does not set a default value, it just doesn't enforce any limit at this point, which means that another limit afterwards may be hit

Reply:
It's ok to rephrase like this? 
`If not set, the container thread pool determines the maximum number of threads the flow can use to optimize the performance when processing messages.`

Reply:
Yeah, the previous sentence was accurate